library/blivet.py

```diff
@@ -1527,7 +1527,7 @@ def _create(self):
         if not self._device:
             members = self._manage_encryption(self._create_members())
             try:
-                pool_device = self._blivet.new_vg(name=self._pool['name'], parents=members)
+                pool_device = self._blivet.new_vg(name=self._pool['name'], parents=members, shared=self._pool['shared'])
```

Comment:
Need some sort of logic here to avoid using the `shared` parameter if not supported

```python
                if self._blivet.new_vg supports 'shared' parameter:
                    pool_device = self._blivet.new_vg(name=self._pool['name'], parents=members, shared=self._pool['shared'])
                else:
                     pool_device = self._blivet.new_vg(name=self._pool['name'], parents=members)
```
There's probably some way to use introspection to see if the `new_vg` method supports `shared`

or some other way to dynamically construct the `new_vg` arguments e.g. `new_vg_args = {}` the pass like `new_vg(**new_vg_args)`

This is what is causing some of the test failures

Reply:
I have modified the condition so the `shared` parameter is not used when its value is default (`false`). This should fix the tests.